packages/cli/medusa-cli/src/reporter/index.ts

```diff
@@ -5,7 +5,7 @@ import { ulid } from "ulid"
 import winston from "winston"
 import { panicHandler } from "./panic-handler"
 
-const LOG_LEVEL = process.env.LOG_LEVEL || "info"
+const LOG_LEVEL = process.env.LOG_LEVEL || "http"
```

Comment:
Is this `http` value intended?
if `env.LOG_LEVEL` is not provided, shouldn't it be `info` for dev and `error` for production?

Reply:
The idea is to keep the current logs intact, so the http logs have to be present and therefore default to http which includes info, warn, error

Reply:
Is this a common log level thoght? usually they are like `fatal, error, warn, info, debug, trace`.

Reply:
Yeah it is from npm ans Winston chooses it as a ref.

Reply:
It is the first time I see this [`http`](https://docs.npmjs.com/cli/v8/using-npm/logging) 🤣 
It doesn't seem to be very standard elsewhere, but this is configurable anyway... so, all good.